include/cuco/detail/static_multimap/static_multimap.inl

```diff
@@ -249,6 +284,22 @@ void static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator,
   impl_->contains_if_async(first, last, stencil, pred, output_begin, ref(op::contains), stream);
 }
 
+template <class Key,
+          class T,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+template <typename InputIt>
+static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::size_type
+static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::count(
+  InputIt first, InputIt last, cuda::stream_ref stream) const noexcept
```

Comment:
`count` needs to copy the result back to the host so there is a CUDA API call which should be wrapped with `CUCO_CUDA_TRY` which could potentially throw. So this function cannot be marked as `noexcept`.

Reply:
Good catch. Addressed.